### PR TITLE
Adding py-cloudfs app to sample repo with pytests

### PR DESCRIPTION
**Pull request by ewucc**
Part of https://github.com/klothoplatform/klotho-history/issues/604

### CODE REVIEW

**Review by yuval-klotho — APPROVED** [review]
lgm other than the node instructions

**Comment by DavidSeptimus-Klotho** [comment]
`tests` needs to be renamed to `test` for the integ test workflow to pick up the policy pack

**Review by DavidSeptimus-Klotho — CHANGES_REQUESTED** [review]
see previous comment

File: py-cloudfs/README.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+run the terminal commands:
+```sh
+# Compile the app
+klotho . --app py-cloudfs -p aws
+# Go into the compiled directory
+cd compiled
+# If you didn't set the aws region as indicated in the compiler output, do that now
+pulumi config set aws:region YOUR_REGION -s py-cloudfs
+# npm install
+npm install
```
Comment (jhsinger-klotho) [inline py-cloudfs/README.md:36]: pip

**Comment by jhsinger-klotho** [comment]
rm .pyc files and __pycache__

File: py-cloudfs/README.md (hunk 1 above)
Comment (ewucc) [reply py-cloudfs/README.md:36]: this is inside the compiled directory tho to install the pulumi stuff.

**Pull request merged**